### PR TITLE
feat: add network text on bitcoin settings top bar

### PR DESCRIPTION
**Pull request by Kodylow**
image

image

image

addresses #320

### CODE REVIEW

File: apps/guardian-ui/src/components/SetConfiguration.tsx [hunk 1]
```diff
@@ -277,7 +277,16 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
             />
           </FormGroup>
         )}
-        <FormGroup icon={BitcoinLogo} title='Bitcoin settings' isOpen={false}>
+        <FormGroup
+          icon={BitcoinLogo}
+          title={
+            <>
+              <span>Bitcoin settings: </span>
```
Comment (okjodom) [inline apps/guardian-ui/src/components/SetConfiguration.tsx:284]: needs translation

File: apps/guardian-ui/src/utils/network.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export const getNetworkIndicator = (network: string, bitcoinRpcUrl: string) => {
```
Comment (okjodom) [inline apps/guardian-ui/src/utils/network.tsx:3]: wondering if we have need for a similar util/ux in gateway UI?

**Review by okjodom — DISMISSED** [review]
ack. ✅

File: apps/guardian-ui/src/utils/network.tsx (hunk 2 above)
Comment (Kodylow) [reply apps/guardian-ui/src/utils/network.tsx:3]: no that one already shows the network, can add in a separate PR if we want it there too

Comment (Kodylow) [reply apps/guardian-ui/src/utils/network.tsx]: added it to the guardian dashboard at least

**Comment by Kodylow** [comment]
image

File: packages/ui/src/network.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export const getNetworkIndicator = (network: string, bitcoinRpcUrl: string) => {
```
Comment (wbobeirne) [inline packages/ui/src/network.tsx:3]: Any reason this was made a function and not a component?

File: packages/ui/src/network.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      if (bitcoinRpcUrl === 'https://mutinynet.com/api') {
+        color = 'red';
+        name = 'Mutinynet';
+      } else {
+        color = 'purple';
+        name = 'Signet';
+      }
+      break;
+    case 'regtest':
+      color = 'skyblue';
```
Comment (wbobeirne) [inline packages/ui/src/network.tsx:26]: This has really rough readability on white, can we pick a darker color?

File: packages/ui/src/NetworkIndicator.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+  switch (network) {
+    case 'bitcoin':
+      color = '#FF9900';
+      name = 'Mainnet';
+      break;
+    case 'testnet':
+      color = '#6BED33';
+      name = 'Testnet';
+      break;
```
Comment (wbobeirne) [inline packages/ui/src/NetworkIndicator.tsx:23]: Seems like these are coming back "Unknown" instead

CleanShot 2023-11-30 at 09 57 40@2x
CleanShot 2023-11-30 at 09 57 35@2x

**Review by wbobeirne — CHANGES_REQUESTED** [review]

**Pull request merged**